src/App.jsx

```diff
@@ -17,10 +18,96 @@ function App() {
     });
   }, []);
 
+
+  function showFiles() {
+    let arrow = document.querySelectorAll(".arrow");
+    for (var i = 0; i < arrow.length; i++) {
+      arrow[i].addEventListener("click", (e) => {
+        let arrowParent = e.target.parentElement.parentElement;//selecting main parent of arrow
+        arrowParent.classList.toggle("showMenu");
+        let folder = document.getElementById("folderIcon");
+        if (folder.className === "bx bx-folder-open") {
+          folder.className = "bx bx-folder";
+        } else {
+          folder.className = "bx bx-folder-open";
+        }
+      });
+    }
+  }
+
+  let parsedData = {};
+
+  function fillData() {
+    let fileList = document.querySelector(".sub-menu");
```

Comment:
It's generally a bad idea to query the DOM in the React world. It's better to generate the DOM based on our data.